<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>连接参数1</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit">
        <property name="text">
         <string>172.18.96.51</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pushButton_Connect">
        <property name="text">
         <string>连接</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>连接参数2</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="text">
         <string>9901</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="pushButton_DisConnect">
        <property name="text">
         <string>VPR_Disconnect</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Index</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_Index"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_RegisterMessage">
      <property name="text">
       <string>RegisterMessage</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_VPR_RegisterCallback">
      <property name="text">
       <string>VPR_RegisterCallback</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_Capture">
      <property name="text">
       <string>VPR_Capture</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_GetPlate">
      <property name="text">
       <string>VPR_GetPlate</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_GetBigImg">
      <property name="text">
       <string>VPR_GetBigImg</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_GetSmallImg">
      <property name="text">
       <string>VPR_GetSmallImg</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_GetBinImg">
      <property name="text">
       <string>VPR_GetBinImg</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_GetAllInfo">
      <property name="text">
       <string>VPR_GetAllVehInfo</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_GetVer">
      <property name="text">
       <string>VPR_GetVer</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>394</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
